### DISCUSSION TITLE
Machine Learning KNN

### DISCUSSION THREAD

**Discussion by Keviniscaiji** [community/community#32988] [Programming Help] [answered]
In ML field, what is the inductive bias present in the KNN classification technology?

**Comment by WangHewei16** [comment, marked as answer]
Hello! Several assumptions define the model selection criteria for the ML algorithm. KNN is based on the assumption that adjacent samples in the feature space belong to the same class. Hope my answer can help you.

**Reply by Keviniscaiji** [reply]
Many thanks
